extensions/patterns/manage_service/playbooks/run_manage_service.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Manage Windows Services
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Manage Windows Service
+      ansible.builtin.include_role:
+        name: infra.windows_ops.windows_manage_service
+      vars:
+        windows_manage_service_name: "{{ service_name }}"  # Set by survey
+        windows_manage_service_state: "{{ service_state }}"  # Set by survey
+        windows_manage_service_start_state: "{{ service_start_state }}"  # Set by survey
```

Comment:
Start state or start mode?

Reply:
This is why we have reviews.. :-)  Thank you so much for catching this!